be/src/exec/broker_scanner.cpp

```diff
@@ -161,6 +193,11 @@ Status BrokerScanner::open_file_reader() {
     if (start_offset != 0) {
         start_offset -= 1;
     }
+    //means first range 
+    if (start_offset == 0){
```

Comment:
if start_offset=1, then in line 194, it will be set to start_offset=0.

it is still first range?

Reply:
Yes, the reason for subtracting 1 is that offset is plus 1 in the FE calculation (bytesperinstance = totalbytes / numinstances + 1), so for the first range  start_ Offset is always 0, and other range will not be 0